File: chap15/C#/Hello/Hello/Class1.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Hello
{
    using Autodesk.AutoCAD.Runtime;
    using Autodesk.AutoCAD.ApplicationServices;
    using Autodesk.AutoCAD.EditorInput;
    public class Class1
    {

        [CommandMethod("Hello",CommandFlags.Modal)]
        public void HelloWorld()
        {
            Editor ed = Application.DocumentManager.MdiActiveDocument.Editor;
            ed.WriteMessage("欢迎进入.NET开发AutoCAD的世界！");
        }
    }
}

File: chap15/VB/Hello/Hello/Class1.vb
Imports Autodesk.AutoCAD.Runtime
Imports Autodesk.AutoCAD.ApplicationServices
Imports Autodesk.AutoCAD.EditorInput

Public Class Class1
    <CommandMethod("Hello")> _
    Public Sub Hello()
        ' 获取当前活动文档的Editor对象，也就是命令行
        Dim ed As Editor = Application.DocumentManager.MdiActiveDocument.Editor
        ' 调用Editor对象的WriteMessage方法在命令行上显示文本
        ed.WriteMessage("欢迎进入.NET开发AutoCAD的世界！")
    End Sub
End Class
